lib/src/components/day_view_components.dart

```diff
@@ -104,6 +109,6 @@ class DayPageHeader extends CalendarPageHeader {
           onTitleTapped: onTitleTapped,
           dateStringBuilder: DayPageHeader._dayStringBuilder,
         );
-  static String _dayStringBuilder(DateTime date) =>
+  static String _dayStringBuilder(DateTime date, [DateTime? secondaryDate]) =>
```

Comment:
Can we use named parameter for `secondaryDate`?
Ex : `static String _dayStringBuilder(DateTime date, {DateTime? secondaryDate})`